java/bcprov/Portfile

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           java 1.0
+
+name                bcprov
+version             1.62
+
+categories          java devel security
+license             MIT
+maintainers         nomaintainer
+platforms           darwin
+supported_archs     noarch
+
+description         A Java implementation of cryptographic algorithms
+long_description    ${description}
+homepage            https://www.bouncycastle.org/java.html
+
+master_sites        https://search.maven.org/remotecontent?filepath=org/bouncycastle/${name}-jdk15on/${version}/
+
+distname            ${name}-jdk15on-${version}
+distfiles           ${distname}.jar
+worksrcdir          ${distname}
+
+checksums           rmd160  288577b36c0385dcd4b9f5614929727f4b039a27 \
+                    sha256  2fa0ab71b154da29ac134097bc6bbacd90987dd4c4005516159e6494d1d52ea2 \
+                    size    4558151
+
+use_configure       no
```

Comment:
Move this `use_configure` line to just above the `build` line to match the sequence of the phases. extract -> configure -> build.

Reply:
Done, sorry for the delay.